src/articulation.js

```diff
@@ -198,6 +198,25 @@ export class Articulation extends Modifier {
     return true;
   }
 
+  static easyScoreHook({ articulations }, note, factory) {
```

Comment:
Instead of `factory` how about passing in the `EasyScore` instance itself, and add accessor `getFactory`. This would future-proof it a bit.